public_html/assets/css/nf-core-dark.css

```diff
@@ -56,6 +56,13 @@ blockquote {
   background-color: #181A1B;
   border-color: rgba(102, 102, 102, 0.13);
 }
+/* .stats_keynumbers .bg-icon {
```

Comment:
Please remove this again if it's not needed..

Reply:
Sorry, this was an artifact while creating the before-after screenshots, see commit 
0f06535.